mappings/net/minecraft/client/gui/widget/container/LayoutSettings.mapping

```diff
@@ -1,9 +1,48 @@
 CLASS net/minecraft/unmapped/C_pjarmpvz net/minecraft/client/gui/widget/container/LayoutSettings
 	METHOD m_aqbyqkmb create ()Lnet/minecraft/unmapped/C_pjarmpvz;
-	CLASS C_efyrmufb LayoutSettingsImpl
+	METHOD m_awwfcuvd setRightPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_djhyilbe setPadding (II)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 horizontal
+		ARG 2 vertical
+	METHOD m_ewcvnloj setAlignment (FF)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 x
+		ARG 2 y
+	METHOD m_gouapxzs alignVerticallyTop ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_hyvejfbf setPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_ietlmaqj alignVerticallyBottom ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_iypkkyhh alignVerticallyCenter ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_kohxcisj setLeftPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_kojqxdlf setTopPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_ofoiqyqp alignHorizontallyCenter ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_pmidjbdw setPadding (IIII)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 left
+		ARG 2 top
+		ARG 3 right
+		ARG 4 bottom
+	METHOD m_qckwecgp get ()Lnet/minecraft/unmapped/C_pjarmpvz$C_efyrmufb;
+	METHOD m_vskvmyqn setVerticalPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_vwciawka setHorizontalPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_vyorthoe setBottomPadding (I)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 padding
+	METHOD m_wtmggfev copy ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_xkbxtfqc setAlignmentY (F)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 y
+	METHOD m_ygbfijsh alignHorizontallyLeft ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_yxhxelpo alignHorizontallyRight ()Lnet/minecraft/unmapped/C_pjarmpvz;
+	METHOD m_zygwkpxf setAlignmentX (F)Lnet/minecraft/unmapped/C_pjarmpvz;
+		ARG 1 x
+	CLASS C_efyrmufb Impl
 		FIELD f_hxiztipo leftPadding I
 		FIELD f_kvjbnbeh bottomPadding I
 		FIELD f_lgffdybo rightPadding I
 		FIELD f_uddwylwg alignmentX F
 		FIELD f_vvwvvykx topPadding I
 		FIELD f_wlrjffdq alignmentY F
```

Comment:
hm, I don't remember what I used on my original mappings; I think I used `horizontalAligment` and `verticalAlignment`...?

Reply:
wait, `alignmentX` and `alignmentY` *are* my names